### PR TITLE
Sort orders and attendees by date

### PR DESCRIPTION
**Pull request by chigby**
See #579 for motivation.

So, this is not quite done yet (I think). But I'm putting the code up here because I can't figure something out with Django's template filters. Specifically the date filter. I'd sort of rather have the date formatting stuff happen in the template, but I wasn't able to get any text to show up when I applied the filter.

If it's not something we care that strongly about, then great. Though it might be worth thinking about adding some date & time specific formatting stuff to `ModelTable` that handles basic human-friendly formatting (e.g. we will probably never want to show microseconds) and maybe also time zone stuff.

### CODE REVIEW

**Comment by melinath** [comment]
+1 on timezone... I think CSV can *probably* safely have "raw" data like microseconds?

File: brambling/utils/model_tables.py [hunk 1]
```diff
@@ -713,3 +727,9 @@ def housing_categories(self, obj):
 
     def balance(self, obj):
         return format_money(obj.balance or 0, self.event.currency)
+
+    def completed_date(self, obj):
+        if obj.completed_date:
+            return obj.completed_date.strftime("%Y-%m-%d %H:%M")
+        else:
+            return "n/a"
```
Comment (melinath) [inline brambling/utils/model_tables.py]: Probably best to capitalize (N/A)

File: brambling/utils/model_tables.py [hunk 2]
```diff
@@ -713,3 +727,9 @@ def housing_categories(self, obj):
 
     def balance(self, obj):
         return format_money(obj.balance or 0, self.event.currency)
+
+    def completed_date(self, obj):
+        if obj.completed_date:
+            return obj.completed_date.strftime("%Y-%m-%d %H:%M")
```
Comment (melinath) [inline brambling/utils/model_tables.py]: This is fine, for now, given that the housing date is handled similarly. But this does need to be wrapped in a timezone context manager, like what I think you did for the finances csv?

Comment (chigby) [inline brambling/utils/model_tables.py]: Right, if we're fine with not having it in the template, then I can do the same thing as with the finances table.

Comment (melinath) [inline brambling/utils/model_tables.py]: Yeah, we're fine not having it in the template.

**Pull request merged**